### PR TITLE
nh-wpi-test

### PR DESCRIPTION
**Pull request by noahheller**

### CODE REVIEW

File: src/main/java/frc/robot/Constants2023.java [hunk 1]
```diff
@@ -61,11 +61,13 @@ public class Constants2023 extends GameConstants {
...
     public static final double STEER_PID_FF_V = 0;//0.8;
     public static final double WHEEL_RADIUS = 0.0508;
+    public static final double ROBOT_RADIUS =  0.4191;
+
 
     public static final double MAX_VELOCITY = 4.0; // 4 meters per second
     public static final double MAX_ANGULAR_SPEED = Math.PI * 6; // 1 rotation per second
-    public static final double ROBOT_WIDTH = 0.5969;
-    public static final double ROBOT_LENGTH = 0.5969;
+    public static final double ROBOT_WIDTH = 0.91440;
```
Comment (batchen1) [inline src/main/java/frc/robot/Constants2023.java:69]: These new numbers don't seem correct. They should represent the width and length between swerve modules.

File: src/main/java/frc/robot/subsystems/swervev2/SwervePosEstimator.java [hunk 2]
```diff
@@ -39,6 +50,9 @@ public SwervePosEstimator(GenericEncodedSwerve frontLeftMotor, GenericEncodedSwe
                         backRightMotor.getPosition(),
                 },
                 new Pose2d());
+        NetworkTableInstance inst = NetworkTableInstance.getDefault();
```
Comment (batchen1) [inline src/main/java/frc/robot/subsystems/swervev2/SwervePosEstimator.java:53]: where is all the code to add vision to poseEstimator? Is this in a different branch?

**Review by batchen1 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/Constants2023.java (hunk 1 above)
Comment (noahheller) [reply src/main/java/frc/robot/Constants2023.java:69]: I thought they were for the total robot width and length. I am going to make a new constant for those values

Comment (noahheller) [reply src/main/java/frc/robot/Constants2023.java:69]: nevermind I thought they were used by path planner, but the values are in settings.json, so they dont use the constants

**Comment by noahheller** [comment]
for some reason pushes are not showing up in this pull request, recreating

**Pull request closed without merging**